Add benchmarks for speedups (#473)
python-kasa/python-kasa: README.md

pip install python-kasa[speedups]
```

> With `[speedups]`, the protocol overhead is roughtly an order of magnitude lower (benchmarks available in bench)

Author: ```suggestion
With `[speedups]`, the protocol overhead is roughly an order of magnitude lower (benchmarks available in bench)
```

> With `[speedups]`, the protocol overhead is roughly an order of magnitude lower (benchmarks available in bench)
> 
> ```
> # python3 bench/benchmark.py
> New parser, parsing 100000 messages took 0.6212750830018194 seconds
> Old parser, parsing 100000 messages took 9.37843908299692 seconds

Reviewer: ```suggestion
With `[speedups]`, the protocol overhead is roughly an order of magnitude lower (benchmarks available in devtools).
```

pip install python-kasa[speedups]
```

With `[speedups]`, the protocol overhead is roughly an order of magnitude lower (benchmarks available in devtools).

> ```

Author: ```suggestion
```
